pkg/virt-launcher/virtwrap/api/converter_test.go

```diff
@@ -403,8 +403,8 @@ var _ = Describe("Converter", func() {
     <chassis></chassis>
   </sysinfo>
   <devices>
-    <interface type="bridge">
-      <source bridge="k6t-eth0"></source>
+    <interface type="ethernet">
+      <source></source>
```

Comment:
The title of the commit is the source attr is replaced by the target. Where is the target (<target dev='tap012345678901' managed='no'/>)?

Reply:
The `target` is computed in the `BindMechanism`, not in the converter. This is a `converter` test, which means you won't see any `target` change here. 

Forced to agree that the commit msg is **very** misleading. Will update it.